src/fpylll/io.pyx

```diff
@@ -11,6 +11,7 @@ from .gmp.mpz cimport mpz_t, mpz_set_si, mpz_set
 from cpython.version cimport PY_MAJOR_VERSION
 from fplll.fplll cimport FT_DEFAULT, FT_DOUBLE, FT_LONG_DOUBLE, FT_DPE, FT_MPFR
 from fplll.fplll cimport ZT_MPZ, ZT_LONG
+from fpylll.numpy import is_numpy_integer
```

Comment:
try cimport here?

Reply:
cimport causes it all to yell, because ```is_numpy_integer``` isn't a Cython function (i.e there's no ```.pxd``` file that contains it as a declaration).